extensions/ohif-i18n/src/index.js

```diff
@@ -6,20 +6,22 @@ import pkg from '../package.json';
 import { debugMode, detectionOptions } from './config';
 import locales from './locales';
 
-let translate;
-
 function addLocales(newLocales) {
   customDebug(`Adding locales ${newLocales}`, 'info');
 
   let resourceBundle = [];
 
-  Object.keys(newLocales).map(key => {
-    Object.keys(newLocales[key]).map(namespace => {
-      const locale = newLocales[key][namespace];
-      resourceBundle.push({ key, namespace, locale });
-      i18n.addResourceBundle(key, namespace, locale, true, true);
-    });
-  });
+  for (const key in newLocales) {
```

Comment:
I don't understand what is different between the Object.keys() loop and the `for in` loop. Is there a reason you did this?

Reply:
I was just trying to test changing to `for ... in` to see if the error was here - it was not - and I ended up leaving it like this. I'm reverting this.